docs/architecture.md

```diff
@@ -9,9 +9,14 @@ _Some incomplete notes_
 * **Driver** - Interfaces with [Chrome Debugging Protocol](https://developer.chrome.com/devtools/docs/debugger-protocol)  ([API viewer](https://chromedevtools.github.io/debugger-protocol-viewer/))
 * **Gatherers** - Uses Driver to collect information about the page. Minimal post-processing.
   * **Artifacts** - output of a gatherer
-* **Audits** - Using the Artifacts as input, Audits evaluate a test and assign pass/fail/scoring.
+* **Audit** - Tests for a single feature/optimization/metric. Using the Artifacts as input, an audit evaluates a test and  Resolves to a score, which may be pass/fail/numeric.  Meta info may contain markdown for rich text (code, links, etc).
```

Comment:
nit: "and resolves to a score"

also do we support an etc for markdown or is it just code and links for now?

Reply:
just code and links. code in title, links in description. and that's it, currently.